ui/raidboss/data/07-dt/trial/zoraal-ja-ex.txt

```diff
@@ -0,0 +1,122 @@
+# 937E - Left sword glowing, safe spot is back+right
+
+# Arena phase
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+3.2 "--sync--" Ability { id: "93A7", source: "Zoraal Ja" } window 3.2,1
+8.0 "--sync--" StartsUsing { id: "9398", source: "Zoraal Ja" } window 8,8
+12.7 "Actualize" Ability { id: "9398", source: "Zoraal Ja" }
+23.0 "Multidirectional Divide (cast)" Ability { id: "93A2", source: "Zoraal Ja" }
+34.0 "Multidirectional Divide (damage)" Ability { id: "93A3", source: "Zoraal Ja" }
+35.0 "Half Full" #Ability { id: "939E|9380", source: "Zoraal Ja" }
+35.0 "Forward Edge/Backward Edge" #Ability { id: "9972|937F", source: "Zoraal Ja" }
+44.2 "Multidirectional Divide (cast)" Ability { id: "93A2", source: "Zoraal Ja" }
+55.2 "Multidirectional Divide (damage)" Ability { id: "93A3", source: "Zoraal Ja" }
+55.3 "Regicidal Rage" #Ability { id: "993C", source: "Zoraal Ja" }
+
+# Tiles phase
+70.5 "Dawn of an Age" Ability { id: "9397", source: "Zoraal Ja" } window 5,5
+84.5 "Vollok" Ability { id: "9357", source: "Zoraal Ja" }
+94.7 "Sync" Ability { id: "9359", source: "Zoraal Ja" }
+103.6 "Chasm of Vollok" Ability { id: "939A", source: "Zoraal Ja" }
+104.1 "Half Full" Ability { id: "939E|9380", source: "Zoraal Ja" }
+112.9 "Greater Gateway" Ability { id: "9381", source: "Zoraal Ja" }
+121.1 "Blade Warp" Ability { id: "935E", source: "Zoraal Ja" }
+129.3 "Forged Track (cast)" Ability { id: "935F", source: "Zoraal Ja" }
+137.5 "Forged Track (damage)" Ability { id: "939D", source: "Fang" }
+137.6 "Fiery Edge/Stormy Edge" #Ability { id: "938[36]", source: "Fang" }
+142.3 "Chasm of Vollok" Ability { id: "9389", source: "Zoraal Ja" }
+147.3 "Actualize" Ability { id: "9398", source: "Zoraal Ja" } window 5,5
+
+# Arena phase
+159.5 "Projection of Triumph" Ability { id: "938A", source: "Zoraal Ja" }
+# TODO: `Fang` actor might actually have a random or potentially empty name because of actor reuse
+169.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+169.8 "Projection of Triumph" Ability { id: "938A", source: "Zoraal Ja" }
+174.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+179.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+184.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+188.1 "Half Full" Ability { id: "939E|9380", source: "Zoraal Ja" }
+188.1 "Forward Edge/Backward Edge" #Ability { id: "9972|937F", source: "Zoraal Ja" }
+189.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+194.8 "Siege of Vollok/Walls of Vollok" #Ability { id: "938B", source: "Fang" }
+198.4 "Actualize" Ability { id: "9398", source: "Zoraal Ja" }
+210.5 "Projection of Turmoil" Ability { id: "9A88", source: "Zoraal Ja" }
+260.4 "Bitter Whirlwind" Ability { id: "993E", source: "Zoraal Ja" }
+263.7 "Bitter Whirlwind" Ability { id: "9940", source: "Zoraal Ja" }
+266.8 "Bitter Whirlwind" Ability { id: "9940", source: "Zoraal Ja" }
```

Comment:
Suggest numbering these.  Is `993E` is the cast and `9940` damage?  If so, maybe parentheticals?

Reply:
It's a three-hit buster. The first hit has a cast bar and therefore a different ID, the other two don't have a cast bar.